### ISSUE TITLE
ProtonMail and extra email addresses

### ISSUE THREAD

**Issue by pavarnos** [keybase/client#12501]
Hi

- is there a way to add another email address that can be verified by keybase
- for example: proton mail, which would link my protonmail PGP public key with keybase.

thanks

**Comment by Avamander** [comment]
Yep, a Protonmail proof for Keybase would be really nice. As both products are really privacy-oriented I'm sure it'd be beneficial for both?
